src/creatures/players/player.cpp

```diff
@@ -1526,9 +1526,7 @@ void Player::onCreatureAppear(Creature* creature, bool isLogin)
 			bed->wakeUp(this);
 		}
 
-		if (isLogin) {
```

Comment:
Removing this check wouldn't trigger this  code each time a creature appear?

Reply:
no, because is inside this if, it's a redundant check
https://github.com/opentibiabr/canary/blob/154b408325981268dba6bd31d9667eaa7ad80db0/src/creatures/players/player.cpp#L1508